pkg/helm/actions/helm_actions.go

```diff
@@ -0,0 +1,11 @@
+package actions
+
+import "helm.sh/helm/v3/pkg/action"
+
+type HelmActions struct {
```

Comment:
Please remove `HelmActions` because we are not adding any additional field.

Reply:
removed